config/routes.rb

```diff
@@ -5,6 +5,8 @@
 	get    'm/:token', to: 'messages#show', as: 'message'
 	delete 'm/:token', to: 'messages#destroy', as: 'destroy_message'
   root to: 'messages#new'
+	get 		'/about' => 'pages#about'
```

Comment:
i'd probably just do one space between `get` and the route name. no need to align them this way.